src/Components/App.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from "react";
+import "./App.css"
+import NewsCard from "./NewsCard"
+import RedditCard from "./RedditCard"
+import TwitterCard from "./TwitterCard"
+import GithubCard from "./GithubCard"
+
+const App = () => {
+    const [render, setRender] = useState('github')
+
+    function renderComponent() {
+        switch(render){
+          case 'twitter': return <TwitterCard/>
+          case 'reddit': return <RedditCard/>
+          case 'news': return <NewsCard/>
+          case 'github': return <GithubCard/>
+          default: return null
+        }
+    }
+    return (
+        <div className = "">
+        <nav className="container">
+          <div className="1">
+          <a onClick = { () => setRender('twitter')} className="navLinks">Twitter</a>
```

Comment:
Since these aren't links and are just triggering JS functions, I'd suggest making them buttons rather than anchor elements